odoo/tools/mail.py

```diff
@@ -77,7 +77,7 @@ def getaddresses(fieldvalues):
      'data-class', 'data-mimetype', 'data-original-src', 'data-original-id', 'data-gl-filter', 'data-quality', 'data-resize-width',
      'data-shape', 'data-shape-colors', 'data-file-name', 'data-original-mimetype',
      'data-attachment-id', 'data-format-mimetype',
-     'data-ai-field',
+     'data-ai-field', 'data-ai-record',
```

Comment:
`res_id`, `member_id`, `view-id`, `link-id`...

```suggestion
     'data-ai-field', 'data-ai-record_id',
```
or
```suggestion
     'data-ai-field', 'data-ai-record-id',